credit_calculator.py

```diff
@@ -28,16 +28,15 @@ def months_until_paid_out(credit_sum, monthly_payment, initial_payment, monthly_
 
 if __name__ == "__main__":
     
-    data=[]
-    for arg in sys.argv:
-        data.append(arg)
-        
-    credit_sum = float(data[1])
-    monthly_payment = float(data[2])
-    
+    parser = argparse.ArgumentParser(description='Arguments required to do the calculations')
+    parser.add_argument('credit_sum', type=float, help='Credit Summary')
+    parser.add_argument('monthly_payment', type=float, help='Monthly Payment')
+
+    args = parser.parse_args()
+
     print("DNB: ")
-    months_until_paid_out(credit_sum, monthly_payment, DNB_INITIAL_PAYMENT, DNB_MONTHLY_PERCENTAGE, DNB_FEE)
+    months_until_paid_out(args.credit_sum, args.monthly_payment, DNB_INITIAL_PAYMENT, DNB_MONTHLY_PERCENTAGE, DNB_FEE)
```

Comment:
Also, you can do `credit_sum, monthly_payment = (args.credit_sum, args.monthly_payment)` and use in all functions `credit_sum` and `monthly_payment`. It would be just cleaner code.

Reply:
Done! Thanks